src/_gettsim/arbeitslosengeld_2/freibeträge.yaml

```diff
@@ -0,0 +1,262 @@
+---
+obergrenze_vermögensgrundfreibetrag:
+  name:
+    de: Maximaler Vermögensgrundfreibetrag
+    en: Maximum of basic allowance
+  description:
+    de: >-
+      § 12 (2) Satz 1 Nr. 1 SGB II. Ausnahmeregelung für Personen, die vor 1948 geboren
+      wurden.
+    en: Differs by birth year.
+  unit: Euros
+  reference_period: null
+  type: dict
+  2005-01-01:
+    1947: 33800
+    1948: 13000
+    1958: 13000
+    1964: 13000
+    reference: Artikel 1 G. v. 23.12.2002 BGBl. I S. 4607.
+  2006-08-01:
+    1947: 33800
+    1948: 9750
+    1958: 9750
+    1964: 9750
+    reference: Artikel 1 G. v. 20.07.2006 BGBl. I S. 1706.
+  2008-01-01:
+    1947: 33800
+    1948: 9750
+    1958: 9900
+    1964: 10050
+    note: Artikel 1 G. v. 20.04.2007 BGBl. I S. 554.
+  2023-01-01:
+    reference: G. v. 20.12.2022 BGBl. I S. 2328.
+    note: Bürgergeld-Gesetz
+vermögensgrundfreibetrag_je_lebensjahr:
+  name:
+    de: Vermögensgrundfreibetrag je Lebensjahr
+    en: Basic allowance per year of life
+  description:
+    de: § 12 (2) Satz 1 Nr. 1 SGB II.  Gestaffelt nach Geburtsjahr
+    en: Differs by year of birth.
+  unit: Euros
+  reference_period: null
+  type: dict
+  2005-01-01:
+    1947: 520
+    1948: 200
+    reference: Artikel 1 G. v. 30.12.2002 BGBl. I S. 4619.
+  2006-08-01:
+    1947: 520
+    1948: 150
+    reference: Artikel 1 G. v. 20.07.2006 BGBl. I S. 1706.
+  2023-01-01:
+    reference: G. v. 20.12.2022 BGBl. I S. 2328.
+    note: Bürgergeld-Gesetz
+schonvermögen_bürgergeld:
+  name:
+    de: Schonvermögen beim Bürgergeld
+    en: Protected wealth for the citizen's income
+  description:
+    de: >-
+      Seit der Einführung des Bürgergelds zum 01.01.2023 gilt ein Vermögensfreibetrag
```

Comment:
Not necessary to fix now, but in the future we should strive to write these texts in a way that they are future-proof...